corehq/ex-submodules/pillowtop/processors/form.py

```diff
@@ -91,7 +85,7 @@ def mark_latest_submission(domain, user_id, app_id, build_id, version, metadata,
         metadata
     )
 
-    if last_submission is None or last_submission.submission_date < received_on_datetime:
+    if last_submission is None or last_submission.submission_date <= received_on_datetime:
```

Comment:
I'm concerned that updating the user for every form submission is going to place a huge load on couch. Can we reduce the write load on couch by only updating the user if:
* submission date is more than X time since recorded last submission (only update once very 15 mins
or
* app build number has changed
or
* CC version has changed

Or take some other strategy to batch updates (possibly similar to Emord's async processing model for UCRs).